go/vt/vtgate/viperconfig.go

```diff
@@ -1,16 +1,50 @@
+/*
```

Comment:
Let's also rename this file to be viper_config.go

Reply:
Okay, they're actually for the other PR, I'll make the changes there!